src/kv/store.h

```diff
@@ -131,6 +131,11 @@ namespace kv
         version = v;
         last_replicated = version;
         term_of_last_version = term;
+
+        if (changes.find(ccf::Tables::SIGNATURES) != changes.end())
```

Comment:
I don't love this separate evaluation of "is this globally committable". In particular it means that in unit tests, we will end up with a different sequence of `last_committable` values on backups than on primaries. I'm not sure what a better solution would be.

Reply:
Yes, I agree that this isn't ideal. I feel like that there's no quick win here and that this need to be part of a bigger refactoring beyond the scope of this PR?